packages/camera/camera_windows/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_windows
 description: A Flutter plugin for getting information about and controlling the camera on Windows.
 repository: https://github.com/flutter/plugins/tree/main/packages/camera/camera_windows
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.2.0
+version: 0.2.0+1
```

Comment:
This should be 0.2.1.

Ref: https://dart.dev/tools/pub/versioning#semantic-versions

> For simplicity’s sake, avoid using + after the version reaches 1.0.0.

Reply:
ACK - changing to 0.2.1.